module/VuFind/src/VuFind/Mailer/Mailer.php

```diff
@@ -132,46 +125,31 @@ public function getNewMessage()
      */
     public function resetConnection()
     {
-        // If the transport has a disconnect method, call it. Otherwise, and in case
-        // disconnect fails, revert to the transport instance clone made before a
-        // connection was made.
-        $transport = $this->getTransport();
-        if (is_callable([$transport, 'disconnect'])) {
-            try {
-                $transport->disconnect();
-            } catch (\Exception $e) {
-                $this->setTransport($this->initialTransport);
-            }
-        } else {
-            $this->setTransport($this->initialTransport);
-        }
+        $this->setTransport($this->initialTransport);
         return $this;
     }
 
     /**
      * Get a blank email message object.
      *
-     * @return Message
+     * @return Email
      */
-    public function getNewBlankMessage()
+    public function getNewBlankMessage(): Email
```

Comment:
Should we deprecate this or getNewMessage() now that they are synonymous?

Reply:
Yes, done!